src/config.ts

```diff
@@ -20,10 +20,32 @@ export const config = {
         password: process.env.RMQ_PASSWORD || 'guest',
         username: process.env.RMQ_USERNAME || 'guest',
     },
+    validator: {
+        user: {
+            id: /^[\w-]+@[\w-]+$/,
+            firstname: {
+                minLength: +(process.env.USER_FIRSTNAME_MIN_LEN || 2),
+                maxLength: +(process.env.USER_FIRSTNAME_MAX_LEN || 15),
+            },
+            lastname: {
+                minLength: +(process.env.USER_LASTNAME_MIN_LEN || 2),
+                maxLength: +(process.env.USER_LASTNAME_MAX_LEN || 15),
+            },
+            email: /^[\w-]+\@[\w-]+\.[\w-]+$/i,
+        },
+    },
     server: {
-        port:  +(process.env.PORT || 3000),
+        port: +(process.env.PORT || 3000),
         name: 'user',
     },
+    router: {
+        getMany: {
```

Comment:
Here you can maybe separate it to 2 different objects:
``` javascript
pagination: {
     startIndex: number,
     endIndex: number,
},
sort: {
    sortOrder: '' | '-',
    sortBy: string
}
```

Reply:
Ok, I changed it.